agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java

```diff
@@ -2,21 +2,46 @@
 
 import io.opentelemetry.OpenTelemetry;
 import io.opentelemetry.auto.api.Config;
+import io.opentelemetry.auto.config.AgentConfig;
+import io.opentelemetry.auto.config.ConfigProvider;
 import io.opentelemetry.auto.instrumentation.api.AgentTracer;
+import io.opentelemetry.auto.tooling.exporter.ExporterConfigException;
+import io.opentelemetry.auto.tooling.exporter.ExporterRegistry;
+import io.opentelemetry.auto.tooling.exporter.SpanExporterFactory;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
 import io.opentelemetry.trace.Tracer;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class TracerInstaller {
+  private static final String EXPORTER = "ota.exporter";
+
   /** Register agent tracer if no agent tracer is already registered. */
   public static synchronized void installAgentTracer() {
     if (Config.get().isTraceEnabled()) {
+      // Try to create an exporter
+      final ConfigProvider config = AgentConfig.getDefault();
+      final String exporter = config.get(EXPORTER);
```

Comment:
using the existing Config class, can this be:

```suggestion
      final String exporter = Config.getSettingFromEnvironment(EXPORTER);
```